README.MD

```diff
@@ -25,7 +25,45 @@ Camino-Messenger-Bot is designed to facilitate communication through various mes
 
 This application depends on the [camino-messenger-protocol](https://github.com/chain4travel/camino-messenger-protocol), which is a separate repository. The protocol defines the protobuf messages and services used for communication between the gRPC server and client.
 
-Currently [release-9](https://github.com/chain4travel/camino-messenger-protocol/releases/tag/release-9) of Camino Messenger Protocol is used.
+Currently [release-10](https://github.com/chain4travel/camino-messenger-protocol/releases/tag/release-10) of Camino Messenger Protocol is used.
+
+## Camino Messenger Account (CM Account)
+
+To run the camino-bot-messenger as intended you will need a **CM Account** with a **bot** added to it and **registered services**.
+
+### Creating the CM account
+If you don't already have a **CM account**, for development purposes a CM Account can be created via [camino messenger contracts](https://github.com/chain4travel/camino-messenger-contracts) `hardhat` tool or via [suite](https://dev.suite.camino.network/create) as explained [here](https://docs.camino.network/partners/partner-config/#step-by-step-configuration-guide).
+
+To create a **CM account** and add the bot to it - you will need **two KYC verified** (test) **wallets** - one for the **CM Account**, and one for the **bot**.
```

Comment:
The bot does not have to be KYC verified, is it?

Reply:
Changed to say that only CM Account wallet has to be KYC verified. 
Switched to prod suite with this addition: 
> [!IMPORTANT]
> Remember to ensure the suite is connected to testnet Columbus if you are doing this for development purposes.